juju/juju
# feat(build): use -facade-group=client in rebuild-schema makefile target

The schemas output for python-libjuju should not have contained any non-client schemas -- see python-libjuju issue
[#1099](https://github.com/juju/python-libjuju/issues/1099). Switching to only the client schemas reduces the amount of generated code in python-libjuju substantially.

python-libjuju now uses client-only schema for the schema files currently checked in (see PR [#1160](https://github.com/juju/python-libjuju/pull/1160)), so it would be nice to land this change for 3.6 to streamline things going forward.


## Checklist




## QA steps



* Move the existing `schema.json` elsewhere.
* Run `make rebuild-schema` and verify that a new `schema.json` file has been generated.
* Run `make build` and verify that the same contents are generated for the `schema.json`.
* Verify that the new `schema.json` contains a subset of the contents of the original (moved) `schema.json`.

## Documentation changes



python-libjuju release instructions will be updated to reflect this change. No changes should be needed for juju documentation.

## Links



python-libjuju issue: https://github.com/juju/python-libjuju/issues/1099

draft PR closed in favour of this PR: https://github.com/juju/juju/pull/18148
